charts/localstack/templates/service.yaml

```diff
@@ -37,6 +37,14 @@ spec:
       {{- if and (or (eq $.Values.service.type "NodePort") (eq $.Values.service.type "LoadBalancer")) (not (empty .Values.service.edgeService.nodePort)) }}
       nodePort: {{ .Values.service.edgeService.nodePort }}
       {{- end }}
+    {{- if .Values.service.dnsService }}
+    - name: dns-tcp
+      containerPort: 53
+      protocol: TCP
+    - name: dns-udp
+      containerPort: 53
+      protocol: UDP
```

Comment:
These specs are missing the `port` which is not optional: https://kubernetes.io/docs/reference/kubernetes-api/service-resources/service-v1/
So this would create an invalid service manifest for everyone with `service.dnsService` enabled.

Reply:
Indeed. I should have tested this ahead. The trivial changes are always the ones that get through :(

I've now done as you've suggested and tested with kind. All looks right to me.

I added the following test values:

```
service:
  dnsService: true
```

and

```
service:
  dnsService: true
  clusterIP: 10.96.100.0
  type: ClusterIP
```

Each was successfully applied to the kind cluster and the service definition looked correct to me.